Qualification/Input/qualification_plan.json

```diff
@@ -48,7 +47,7 @@
     },
     {
       "Id": "Caffeine-Fluvoxamine-DDI",
-      "Path": "https://raw.githubusercontent.com/esqLABS/Fluvoxamine-Caffeine-DDI/altv1.1/Fluvoxamine-Caffeine-DDI.json",
+      "Path": "https://raw.githubusercontent.com/Open-Systems-Pharmacology/Fluvoxamine-Caffeine-DDI/alt_v1.0/Fluvoxamine-Caffeine-DDI.json",
```

Comment:
I created "alternative" branch in the OSP repo, copied the files from the ESQLAbs-Repo and created the alt-Release.